.github/workflows/driver.python.documentation.yml

```diff
@@ -11,21 +11,31 @@ on:
         description: Python version used for docs build
         type: string
         default: 3.11
+      use-project-requirements:
+        description: New python packages will contain the requirements for building the documentation in the [docs] section of project dependencies
+        type: boolean
+        default: false
 
 jobs:
   build_pages:
     runs-on: ${{ inputs.run-environment }}
+    if: not ${{inputs.use-project-requirements}}
```

Comment:
this check should probably be removed as it disables the whole job instead of only selecting the correct step for the configuration